### ISSUE TITLE
No results

### ISSUE THREAD

**Issue by LemonLov** [nothinglo/NISwGSP#37]
My OpenCV version is 3.4.14, because it gives me the "No LineSegmentDetector error" when I execute the exe file, so I use FastLineDetector instead, this time no error accur, but there was no results in "0_results" dir. Just feature_pairs-sRANSAC-09-10-398.jpg type image in "1_debug" dir.

**Comment by Sad-Snow** [comment]
yes, it will throw an error when use high vision OpenCV in 3.x.x, cause of no "LineSegmentDetector " class, use lower or even higher vision OpenCV should be fine, like 3.1.0 or 4.x.x+.
your problem seems to be incorrect process of line detection, the output in "1_debug" dir should also have pictures named "line-result-... .jpg".
![1626850348(1)](https://user-images.githubusercontent.com/64819768/126445638-1539e15d-0bcf-4c8b-829e-75192ff21c83.jpg)

**Comment by one123455** [comment]
Have you solved this problem? How to modify the original code?
